src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1376,5 +1375,21 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
         }
 
         #endregion
+
+        internal override NamedTypeSymbol AsNativeInt(bool asNativeInt)
+        {
+            Debug.Assert(this.SpecialType == SpecialType.System_IntPtr || this.SpecialType == SpecialType.System_UIntPtr);
+
+            return asNativeInt ?
+                (NamedTypeSymbol)new NativeIntegerTypeSymbol(this) :
+                this;
+        }
+
+        internal override bool Equals(TypeSymbol t2, TypeCompareKind comparison, IReadOnlyDictionary<TypeParameterSymbol, bool> isValueTypeOverrideOpt = null)
```

Comment:
>Equals [](start = 31, length = 6)

GetHashCode?

Reply:
`GetHashCode()` would simply delegate to `base`.

---
In reply to: [370303898](https://github.com/dotnet/roslyn/pull/40994#discussion_r370303898) [](ancestors = 370303898)